src/sentry/integrations/github/client.py

```diff
@@ -108,18 +108,21 @@ def get_tree(self, repo_full_name: str, tree_sha: str) -> JSONData:
                     f"The tree for {repo_full_name} has been truncated. Use different a approach for retrieving contents of tree."
                 )
             tree = contents["tree"]
-        except ApiError as e:
-            json_data: JSONData = e.json
-            msg: str = json_data.get("message")
```

Comment:
Sometimes APIError is raised with a json objects and sometimes just a string.

This fixes [this error](https://sentry.io/organizations/sentry/discover/sentry:be11666d210c42a98bc4d71d0825effc/?field=title&field=release&field=environment&field=user.display&field=timestamp&name=AttributeError%3A+%27NoneType%27+object+has+no+attribute+%27get%27&project=1&query=issue%3ASENTRY-WSE&sort=-timestamp&statsPeriod=16h&yAxis=count%28%29).
<img width="305" alt="image" src="https://user-images.githubusercontent.com/44410/208921590-446923e9-d32c-48c3-b690-93e68a6cce61.png">